README.md

```diff
@@ -9,15 +9,15 @@
 [foundry]: https://getfoundry.sh
 [foundry-badge]: https://img.shields.io/badge/Built%20with-Foundry-FFDB1C.svg
 
-This repository contains the core smart contracts of the Sablier V2 Protocol. For higher-level logic, see the
+This repository contains the core smart contracts of the Sablier Lockup Protocol. For higher-level logic, see the
 [sablier-labs/v2-periphery](https://github.com/sablier-labs/v2-periphery) repository.
 
 In-depth documentation is available at [docs.sablier.com](https://docs.sablier.com).
 
 ## Background
 
-Sablier V2 is a token distribution protocol used by DAOs and businesses for vesting, payroll, airdrops, and more. Our
-flagship model is the linear stream, which distributes assets on a continuous, by-the-second basis.
+Sablier Lockup Protocol is a token distribution protocol used by DAOs and businesses for vesting, payroll, airdrops, and
```

Comment:
should we remove "payroll"

Reply:
Not sure. Lockup can be used for payroll as well 😅

Reply:
agree